lib/checkother.h

```diff
@@ -215,6 +216,8 @@ class CPPCHECKLIB CheckOther : public Check {
 
     void checkComparePointers();
 
+    void checkModuloOfOne();
```

Comment:
This header needs 2 additional tweaks:
 * Please ensure that the warning is reported in `getErrorMessages(ErrorLogger *errorLogger, const Settings *settings)`
 * Please write a short description in `classInfo()`